libr/core/rtr.c

```diff
@@ -1847,6 +1847,8 @@ R_API int r_core_rtr_cmds (RCore *core, const char *port) {
 	}
 
 	s = r_socket_new (0);
+	s->local = r_config_get_i(core->config, "tcp.islocal");
```

Comment:
Missing space before (

Reply:
Uhm, why not reuse http.bind for this? Tcp. Is not an eval prefix used anywhere, would be good to reuse something i think

Reply:
```c
libr/core/rtr.c-444-                            s->local = true;
libr/core/rtr.c-445-                    } else if (!strcmp (host, "localhost")) {
libr/core/rtr.c-446-                            s->local = true;
libr/core/rtr.c-447-                    } else if (!strcmp (host, "127.0.0.1")) {
libr/core/rtr.c-448-                            s->local = true;
libr/core/rtr.c-449-                    } else if (!strcmp (host, "local")) {
libr/core/rtr.c-450-                            s->local = true;
libr/core/rtr.c:451:                            r_config_set (core->config, "http.bind", "localhost");
libr/core/rtr.c-452-                    } else if (host[0]=='0' || !strcmp (host, "public")) {
libr/core/rtr.c-453-                            // public
libr/core/rtr.c-454-                            host = "127.0.0.1";
libr/core/rtr.c:455:                            r_config_set (core->config, "http.bind", "0.0.0.0");
libr/core/rtr.c-456-                            s->local = false;
libr/core/rtr.c-457-                    } else {
libr/core/rtr.c-458-                            s->local = true;
libr/core/rtr.c-459-                    }
libr/core/rtr.c-460-            } else {
libr/core/rtr.c-461-                    s->local = true;
libr/core/rtr.c-462-            }

```

Reply:
Like the logic above?

Reply:
Take http server, it's restricted. I tried to run some commands, and it blocks them.
TCP server works, and it's a separate implementation. I don't know when http will be ok.
Adding a separate namespace for rtr(tcp based) is fine, isn't it?
http.tcp.bind? is it going to better?